austation/code/modules/power/fuel_rods.dm

```diff
@@ -88,6 +108,20 @@
 						N.fuel_rods += D
 						qdel(src)
 
+			if("bluespace") // telecrystal rod turns into processed telecrystal rod
+				if(!grown)
+					name = "Fully Grown Bluespace Crystal Fuel Rod"
+					desc = "A heavy-duty low-grade fuel rod which fissiles much slower than standard counterparts. Its growth stage has finished, and its power now starts to wane. You also can harvest bluespace crystals which should've been multiplied by now."
+					if (rad_strength > 200)
```

Comment:
Why are you checking rad strength? This is only supposed to be used for radiation pulses

Reply:
Read code why I am doing this. If you want me to further complicate the `/obj/item/twohanded/required/fuel_rod` class, then, sure, I can remove this.